bam_readcount_helper.py

```diff
@@ -16,8 +16,14 @@ def generate_region_list(hash):
     fh.close()
     return fh.name
 
-def filter_sites_in_hash(region_list, bam_file, ref_fasta, sample, output_dir, insertion_centric):
-    bam_readcount_cmd = ['/usr/bin/bam-readcount', '-f', ref_fasta, '-l', region_list, '-w', '0', '-b', '20']
+def filter_sites_in_hash(region_list, bam_file, ref_fasta, sample, output_dir, insertion_centric, map_qual, base_qual):
+    bam_readcount_cmd = ['/usr/bin/bam-readcount', '-f', ref_fasta, '-l', region_list, '-w', '0', '-b']
+    if base_qual == -1: #default value in bam_readcount.cwl
+        bam_readcount_cmd.append('20')
+    else:
+        bam_readcount_cmd.append(str(base_qual))
+    if map_qual != -1: #default value in bam_readcount.cwl
```

Comment:
It might be nicer to have these be optional parameters instead of passing a special value.  (Maybe we're at the point of wanting this helper to use `argparse`?)

Until we get better versioning of Docker images in our pipelines, we may need to revise existing processing profiles that include this workflow step if we break backwards compatibility with the current version of the helper.